ledger-api/grpc-definitions/com/daml/ledger/api/v1/active_contracts_service.proto

```diff
@@ -61,4 +61,8 @@ message GetActiveContractsResponse {
   // The list of contracts that were introduced by the workflow with ``workflow_id`` at the offset.
   // Must be a valid LedgerString (as described in ``value.proto``).
   repeated CreatedEvent active_contracts = 3;
+
```

Comment:
IIRC @moritzkiefer-da wanted to specify the desired offset also in the request. Do you plan to add this to `GetActiveContractsRequest` as an optional field?

Reply:
Another comment: Do we still need to retain the `ledger_id` field? Could we just make it `reserved` in 3.0?

Reply:
Added comments where applicable